src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java

```diff
@@ -198,10 +198,25 @@ public enum SlimefunTag implements Tag<Material> {
     PICKAXE_OF_THE_SEEKER_BLOCKS,
 
     /**
-     * All materials which the {@link SmeltersPickaxe} will try and smelt.
+     * All the materials that can pickaxe smelt in {@link SmeltersTool}.
      */
     SMELTERS_PICKAXE_BLOCKS,
 
+    /**
+     * All the materials that can shovel smelt in {@link SmeltersTool}.
+     */
+    SMELTERS_SHOVEL_BLOCKS,
+
+    /**
+     * All the materials that can axe smelt in {@link SmeltersTool}.
+     */
+    SMELTERS_AXE_BLOCKS,
```

Comment:
Can we remove this now? Is there a benefit to having 3 tags over just 1?

Reply:
nope ig